axis.go

```diff
@@ -373,7 +373,9 @@ func (DefaultTicks) Ticks(min, max float64) (ticks []Tick) {
 	}
 	majorDelta := float64(majorMult) * tens
 	val := math.Floor(min/majorDelta) * majorDelta
-	prec := maxInt(precisionOf(min), precisionOf(max))
+	// Precision required is not just to see a value for the ticks, but to
+	// measure and distinguish the tick points from each other accurately
```

Comment:
Please delete the comment. The code tells us what is going on here.

Also, maxInt needs to be deleted.